cli/src/commands/working-groups/verifyValidator.ts

```diff
@@ -0,0 +1,32 @@
+import { flags } from '@oclif/command'
+import WorkingGroupsCommandBase from '../../base/WorkingGroupsCommandBase'
+import { WorkingGroups } from '../../Types'
+
+export default class VerifyValidatorAccountCommand extends WorkingGroupsCommandBase {
+  static description = 'Membership lead/worker verifies validator membership profile'
+  static flags = {
+    memberId: flags.string({
+      required: true,
+      description: 'Membership ID of the validator to verify',
+    }),
+
+    ...WorkingGroupsCommandBase.flags,
+  }
+
+  async run(): Promise<void> {
+    const { memberId } = this.parse(VerifyValidatorAccountCommand).flags
+
+    const nowGroup = this.group;
+
+    await this.setPreservedState({ defaultWorkingGroup: WorkingGroups.Membership })
+    const worker = await this.getRequiredWorkerContext()
+
+    if (!worker) {
+      this.error('Only membership WG lead/worker can perform this command')
+    } else {
+      this.getOriginalApi().tx.membershipWorkingGroup.workerRemark(Number(worker), memberId || "")
```

Comment:
The second argument here is supposed to be the serialized protobuf message, not a memberId.